lib/puppet/type/sensu_configure.rb

```diff
@@ -35,11 +35,6 @@
     desc "Password to use with sensuctl configure"
   end
 
-  newparam(:bootstrap_password) do
```

Comment:
Changing the public interface feels like a breaking change.

Reply:
If anyone ever changed this value then it would have resulted in errors. The default value was the only value that would work as it's the Sensu Go default admin password. Also we do not document the use of `sensu_configure` as it's kind of a private type that we use internally in the backend class and now moved to the cli class. If anyone changed this value then their install would have not worked so I think it's safe to consider this backwards compatible.